tools/coav-control/coav-control.sh.in

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:        Collision Avoidance Daemon
+# Required-Start:  $network #FIXME
+# Required-Stop:   $network #FIXME
+# Default-Start:   2 3 4 5
+# Default-Stop:
+# Description:     Enable collision avoidance capabilities.
+### END INIT INFO
+
+. /etc/init.d/functions
+
+get_pid() {
+    pidof "coav-control"
+}
+
+start() {
+    nohup @CMAKE_INSTALL_FULL_BINDIR@/coav-control \
+        -d DI_OBSTACLE \
+        -a QC_STOP \
+        -s ST_REALSENSE \
```

Comment:
is there any elegant solution to not have this hardcoded? Config support in coav-control maybe?